### PR TITLE
fix(fxa-auth-server): change email template

### LINKED ISSUE #8011: Update copy in passwordChangeRequired email template
## Description

There are a couple areas we can improve the copy in [passwordChangeRequired](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/templates/passwordChangeRequired.html) email template.

1) Add a an header (h1, refer to other templates to see how it's done) titled "Password Change Required"
2) In the line that says, "Important: pick a different password", capitalize "Pick"
3) Change the line that says "The Firefox accounts team" to "the Firefox Accounts team"

Ensure these copy changes are reflected in the [text](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/templates/passwordChangeRequired.txt#L11:L11) version as well.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-3278)
┆Issue Number: FXA-3278

### PR DESCRIPTION
**Pull request by karinabk**
## This pull request

- Improves the styling of the email template (passwordChangeRequired)

## Issue that this pull request solves

Closes #8011

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

### CODE REVIEW

File: packages/fxa-auth-server/lib/senders/templates/passwordChangeRequired.txt [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 {{{t "We detected suspicious behavior on your Firefox account. To prevent unauthorized access to your Firefox Account, we’ve disconnected all devices on your account and are requiring you to change your password as a precaution." }}}
 
 {{{t "Sign back into any device or service where you use your Firefox account and follow the steps that will be presented to you." }}}
 
-{{{t "Important: pick a different password than what you were previously using and make sure that it is different from your email account." }}}
+{{{t "Important: Pick a different password than what you were previously using and make sure that it is different from your email account." }}}
 
 {{{t "Best," }}}
-{{{t "The Firefox Accounts team" }}}
+{{{t "the Firefox Accounts team" }}}
```
Comment (jodyheavener) [inline packages/fxa-auth-server/lib/senders/templates/passwordChangeRequired.txt:10]: Oh whoops, I see I noted in the issue that "the" should be lowercase, but I didn't realize it was at the beginning of the sentence. These "the"s you changed can be reverted back to "The"s.

**Comment by jodyheavener** [comment]
@karinabk Looks great, and thanks for making that change. I think the only thing left is to squash your two commits into a single commit, then we should be good to merge.

**Comment by karinabk** [comment]
> @karinabk Looks great, and thanks for making that change. I think the only thing left is to squash your two commits into a single commit, then we should be good to merge.

@jodyheavener  done

**Review by jodyheavener — APPROVED** [review]
Nice job! Thanks for this.

**Pull request merged**